src/ServiceControl/NewFeature/Web/NewFeatureController.cs

```diff
@@ -0,0 +1,17 @@
+namespace ServiceControl.NewFeature.Web;
+
+using Microsoft.AspNetCore.Mvc;
+using Persistence.NewFeature;
+
+[ApiController]
+[Route("api")]
+public class NewFeatureController(INewFeatureDataStore newFeatureDataStoreDataStore)
```

Comment:
Here we are declaring a new controller, this interesting thing is that this would be in the "default" `AssemblyLoadContext`